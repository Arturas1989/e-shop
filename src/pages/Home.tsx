export const Home = () => {
  return (
    <main data-testid='home'>Home</main>
  )
};
