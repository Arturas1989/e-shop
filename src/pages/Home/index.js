export { Hero } from './components/Hero/Hero';
export { FeaturedProducts } from './components/FeaturedProducts/FeaturedProducts';
export { Testimonials } from './components/Testimonials/Testimonials';