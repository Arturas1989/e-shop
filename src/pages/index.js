export { Home } from './Home/Home';
export { ProductsPage } from './Products/ProductsPage';
export { PageNotFound } from './PageNotFound/PageNotFound';

export { Register } from './Auth/Register/Register';
export { Login } from './Auth/Login/Login';
