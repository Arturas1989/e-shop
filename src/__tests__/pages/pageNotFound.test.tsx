import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { PageNotFound } from '../../pages';

describe('footer components renders', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(
        <BrowserRouter>
          <PageNotFound />
        </BrowserRouter>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});