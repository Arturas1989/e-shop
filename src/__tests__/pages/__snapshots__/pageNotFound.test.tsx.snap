// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageNotFound components renders renders correctly 1`] = `
<main
  className="page-not-found"
>
  <h1>
    404
  </h1>
  <img
    alt="page not found"
    src="Page not found.png"
  />
  <h2>
    Page not found
  </h2>
  <p>
    We're sorry, the page you requested could not be found
  </p>
  <button
    className="primary-button"
  >
    <a
      href="/"
    >
      Go to home page
    </a>
  </button>
</main>
`;
